diff --git a/python-zope-event.spec b/python-zope-event.spec
index 7071057..799477d 100644
--- a/python-zope-event.spec
+++ b/python-zope-event.spec
@@ -1,3 +1,5 @@
+%global _without_docs 1
+%global _with_bootstrap 1
 # Running the tests requires python3-zope-testrunner, which requires
 # python3-zope-interface, which requires this package.  Build in bootstrap
 # mode to avoid the circular dependency.
