diff --git a/python-sphinx.spec b/python-sphinx.spec
index fd0c7e6..a3bf722 100644
--- a/python-sphinx.spec
+++ b/python-sphinx.spec
@@ -1,3 +1,5 @@
+%global _without_sphinxcontrib 1
+%global _without_tests 1
 # When bootstrapping sphinx in Fedora, we don't yet have sphinxcontrib-*
 # Without the packages, we have warnings in docs, but it's not a hard dependency
 # We don't want to support sphinxcontrib-* in RHEL, hence disabling the dependencies
