diff --git a/python-snakemake-interface-executor-plugins.spec b/python-snakemake-interface-executor-plugins.spec
index abb3ebd..c272bbe 100644
--- a/python-snakemake-interface-executor-plugins.spec
+++ b/python-snakemake-interface-executor-plugins.spec
@@ -1,3 +1,4 @@
+%global _with_bootstrap 1
 # Work around a circular dependency on snakemake and on
 # python-snakemake-executor-plugin-cluster-generic.
 %bcond bootstrap 0
