commit c8594858c775d50dde3da705662ada391943eae1
Author: Miro Hrončok <miro@hroncok.cz>
Date:   Sat Jul 1 17:32:03 2023 +0200

    Temporary shortcut for the Python 3.12 rebuild

diff --git a/python-vcrpy.spec b/python-vcrpy.spec
index 8fb8507..e3c218d 100644
--- a/python-vcrpy.spec
+++ b/python-vcrpy.spec
@@ -22,10 +22,10 @@ BuildRequires:      python3dist(pytest)
 # For checking imports.
 # https://vcrpy.readthedocs.io/en/latest/installation.html
 BuildRequires:      python3dist(aiohttp)
-BuildRequires:      python3dist(boto)
-BuildRequires:      python3dist(boto3)
+#BuildRequires:     python3dist(boto) -- not ready yet for Python 3.12
+#BuildRequires:     python3dist(boto3) -- not ready yet for Python 3.12
 BuildRequires:      python3dist(httplib2)
-BuildRequires:      python3dist(httpx)
+#BuildRequires:     python3dist(httpx) -- not ready yet for Python 3.12
 BuildRequires:      python3dist(requests)
 BuildRequires:      python3dist(tornado)
 %endif
@@ -67,7 +67,7 @@ Summary:            %{summary}
 
 
 %check
-%pyproject_check_import
+%pyproject_check_import -e vcr.stubs.boto_stubs -e vcr.stubs.httpx_stubs
 
 %if %{with tests}
 # These tests make lots of outgoing connections, so we can't run them in
