diff --git a/python-sphinx-theme-builder.spec b/python-sphinx-theme-builder.spec
index e0d2df5..f197fed 100644
--- a/python-sphinx-theme-builder.spec
+++ b/python-sphinx-theme-builder.spec
@@ -1,3 +1,4 @@
+%global _with_bootstrap 1
 # This package serves both as a build backend and a tool for theme developers.
 # During the Python bootstrap we need the build functionality for python-furo.
 # cli extra, needed for the application usage, has got a long chain of
