diff --git a/python-fsspec.spec b/python-fsspec.spec
index aef28a0..9b3ad91 100644
--- a/python-fsspec.spec
+++ b/python-fsspec.spec
@@ -1,3 +1,4 @@
+%global _with_bootstrap 1
 # Avoid dependency loops:
 #     fsspec -> distributed -> dask -> fsspec
 #     fsspec -> gcsfs -> fsspec
@@ -27,10 +28,12 @@ BuildRequires:  python3dist(cloudpickle)
 %if %{without bootstrap}
 #BuildRequires:  python3dist(distributed) -- not yet available in Fedora
 BuildRequires:  python3dist(zarr)
+# notebook does not need bootstrap
+# it was moved here to expedite the Python 3.13 rebuild
+BuildRequires:  python3dist(notebook)
 %endif
 BuildRequires:  python3dist(jinja2)
 BuildRequires:  python3dist(lz4)
-BuildRequires:  python3dist(notebook)
 BuildRequires:  python3dist(numpy)
 BuildRequires:  python3dist(python-snappy)
 BuildRequires:  python3dist(zstandard)
